src/Symfony/Bundle/FrameworkBundle/composer.json

```diff
@@ -47,7 +47,7 @@
         "symfony/form": "~2.8|~3.0",
         "symfony/expression-language": "~2.8|~3.0",
         "symfony/process": "~2.8|~3.0",
-        "symfony/serializer": "~2.8|^3.0",
+        "symfony/serializer": "~3.2",
```

Comment:
I agree. Maybe merging the new PR introducing this change before this one will reduce the amount of work for @tgalopin.